neo.UnitTests/SmartContract/UT_Syscalls.cs

```diff
@@ -1,66 +1,197 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
 using Neo.SmartContract;
 using Neo.VM;
+using Neo.VM.Types;
 using System.Linq;
 
 namespace Neo.UnitTests.SmartContract
 {
     [TestClass]
     public class UT_Syscalls
     {
+        [TestMethod]
+        public void System_Blockchain_GetBlock()
+        {
+            var tx = new Transaction()
+            {
+                Script = new byte[] { 0x01 },
+                Attributes = new TransactionAttribute[0],
+                Cosigners = new Cosigner[0],
+                NetworkFee = 0x02,
+                SystemFee = 0x03,
+                Nonce = 0x04,
+                ValidUntilBlock = 0x05,
+                Version = 0x06,
+                Witnesses = new Witness[] { new Witness() { VerificationScript = new byte[] { 0x07 } } },
+                Sender = UInt160.Parse("FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"),
```

Comment:
I think that we discussed that, could not be the `Sender` just a flag of the cosigners.

Reply:
https://github.com/neo-project/neo/issues/1043

Reply:
When #1043 will be design should change all affected parts.

Reply:
Ok, I just mentioned for us to remember that we can also move in that way.